tests/dialects/test_transform.py

```diff
@@ -1,250 +0,0 @@
-from conftest import assert_print_op
```

Comment:
Is this used anywhere else?

Reply:
Too many places, I've got some branches locally that remove this dependency, which is kinda what motivated this PR in the first place.